.github/actions/install-end2end-dependencies/action.yaml

```diff
@@ -76,6 +76,9 @@ runs:
           sudo chmod +x /usr/local/bin/tilt
           sudo curl -sSL https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F${{ env.KUSTOMIZE_VERSION }}/kustomize_${{ env.KUSTOMIZE_VERSION }}_linux_amd64.tar.gz | sudo tar -xvz
           sudo install kustomize /usr/local/bin
+          sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://dl.k8s.io/apt/doc/apt-key.gpg
```

Comment:
Do we still need that if we use `Azure/setup-kubectl@v3` now?

Reply:
I'll remove it and see